docs/sources/next/results-output/real-time/opentelemetry.md

```diff
@@ -0,0 +1,82 @@
+---
+title: 'OpenTelemetry'
+description: 'Use the OpenTelemetry output to send test results in OTEL metrics format.'
+weight: 00
+---
+
+# OpenTelemetry
+
+k6 can push test run metrics in a [OpenTelemetry metrics format](https://opentelemetry.io/docs/specs/otel/metrics/) to a OTEL metrics collector or a metrics backend that support OTEL metrics fromat by using the [xk6-output-opentelemetry extension](https://github.com/grafana/xk6-output-opentelemetry/).
+
+This is work-in-progress extension, and we are looking for feedback and contributions. Feel free to open an issue directly in the [grafana/xk6-output-opentelemetry repository](https://github.com/grafana/xk6-output-opentelemetry/issues).
```

Comment:
```suggestion
This is an experimental extension, and we are looking for feedback and contributions. Feel free to open an issue directly in the [grafana/xk6-output-opentelemetry repository](https://github.com/grafana/xk6-output-opentelemetry/issues).
```

Reply:
I'd avoid using an experimental extension term until it becomes an experimental extension in k6 (hopefully v0.53) to avoid potential misunderstandings. Do you think this is a valid concern?

Reply:
```suggestion
This is an extension, and we are looking for feedback and contributions and aiming to graduate it as an experimental output within k6 core in one of the future releases. Feel free to open an issue directly in the [grafana/xk6-output-opentelemetry repository](https://github.com/grafana/xk6-output-opentelemetry/issues).
```
The part that I don't like is the fact we mark it as a work in progress which isn't anymore the reality, AFAIK. They can use and trust it.

I made a new suggestion, anyway, feel free to reject it if you don't like it.